.travis.yml

```diff
@@ -1,9 +1,18 @@
 sudo: false
 language: scala
-scala:
-- 2.11.7
 jdk:
-- oraclejdk8
+  - openjdk8
+
+matrix:
```

Comment:
an alternative could be: `script: PLAY_VERSION=2.5 sb +test; PLAY_VERSION=2.6 sb +test;`
to avoid repeating the cross scala versions?